src/widgets/places/places.ts

```diff
@@ -133,6 +133,16 @@ const placesWidget: WidgetFactory<{}, PlacesWidgetOptions> = (
         .setQueryParameter('aroundLatLngViaIP', false)
         .setQueryParameter('aroundLatLng', position || undefined);
     },
+
+    getRenderState(renderState) {
```

Comment:
this is all right, but looking at this file, I can see it's setting parameters in init, isn't that duplicate with getWidgetSearchParameters? can probably just be done in a separate PR

Reply:
Hmm. If I understand correctly, when `placesAutocomplete.on('change'` happens in `init`, it should store the information somewhere else and let `getWidgetSearchParameters` read it and return the updated search parameters, right?
But how does 'change' event listener of placesAutocomplete trigger a re-render?

Reply:
Let's leave it out of this PR and think about it another time